packages/dropdown/README.md

```diff
@@ -0,0 +1,26 @@
+# Dropdown
+
+A dropdown toggles an overlay menu of actions. It appears as a result of a user interacting with some trigger (usually a button or icon) to open the menu.
+
+Dropdown menus can be opened via cursor or via keyboard navigation, and it's menu items can navigated and selected via cursor or via keyboard navigation.
+
+## Dropdown triggers
+By default, use one of the dropdown button components (e.g.: `PrimaryDropdownButton`) as the dropdown trigger.
```

Comment:
I know i'm questioning stuff outside this PR, but: What's the reasoning to go with very specific components (like `PrimaryDropdownButton`) instead of making them parameterized with e.g. an appearance?

Appearance could even be stringly-typed so that users would not have to import the `DropdownButtonAppearance`-enum.

```
<DropdownButton appearance="secondary">Edit</DropdownButton>
```

That's a very opinionated question and i'm totally fine with a "🤷‍♂".

Reply:
We chose to do buttons this way to simplify the prop interface.